src/app/admin/[date]/page.tsx

```diff
@@ -230,7 +231,7 @@ const CreateMenu = ({ params }: { params: { date: string } }) => {
     <div>
       {contextHolder}
       <Title className="text-3xl sm:text-4xl mb-12 font-black text-[#1F2937]">
-        Menu {params.date}
+        {new Date(params.date).toLocaleDateString(pickerLocale.lang.locale, { weekday: 'long', month: 'numeric', day: 'numeric'})}
```

Comment:
Pls udělej z toho promněnou, takhle to není readable
```suggestion
        {dateTitle}
```